lakefs_provider/hooks/lakefs_hook.py

```diff
@@ -30,19 +30,21 @@ class LakeFSHook(BaseHook):
         access_key_id, secret_access_key and lakeFS server endpoint.
     :type lakefs_conn_id: str
     """
+    conn_name_attr = "lakefs_conn_id"
+    default_conn_name = "lakefs_conn_id"
+    conn_type = "lakefs"
+    hook_name = "lakefs"
 
     def __init__(self, lakefs_conn_id: str) -> None:
         super().__init__()
         self.lakefs_conn_id = lakefs_conn_id
 
     def get_conn(self) -> LakeFSClient:
         conn = self.get_connection(self.lakefs_conn_id)
-
         configuration = lakefs_client.Configuration()
         configuration.username = conn.extra_dejson.get("access_key_id", None)
```

Comment:
I have  removed "extra" field and added secret and access key in airflow UI. Tested, it is working fine.
I wonder why somone will now create a HTTP connection knowing there exist a lakeFS connection with the airflow after this release.

Reply:
I have added the "lakefs" as connection type in provider workflow and the workflow passed successfully.  I think we should create seprate request to change in the airflow docs rather than again adding support in lakefs_provider hook.

Reply:
@vaibhavk1992, I'm talking about existing connections - not new ones.